frontend/app/controllers/TierController.scala

```diff
@@ -38,8 +38,8 @@ trait DowngradeTier extends ActivityTracking {
     val futureTierName = "Friend"
     for {
       subscriptionStatus <- subscriptionService.getSubscriptionStatus(request.member)
-      currentSubscription <- subscriptionService.getSubscriptionDetails(subscriptionStatus.current)
-      futureSubscription <- subscriptionService.getSubscriptionDetails(subscriptionStatus.future.get)
+      currentSubscription <- subscriptionService.getSubscriptionDetails(subscriptionStatus.currentVersion)
+      futureSubscription <- subscriptionService.getSubscriptionDetails(subscriptionStatus.futureVersionOpt.get)
```

Comment:
Are we using xxxOpt (i.e. futureVersionOpt) Hungarian notation elsewhere to identify types. Why not just futureVersion?